Source/SwiftLintFramework/Rules/Lint/HashFunctionRule.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import SourceKittenFramework
+
+public struct HashFunctionRule: ASTRule, ConfigurationProviderRule, AutomaticTestableRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "hash_function",
+        name: "Hash Function",
+        description: "The new hash function should be preferred in general. " +
```

Comment:
Maybe "Prefer using implementing `hash(into:)` instead of `hashValue.`?

Reply:
My English is not that good, but should it not be "Prefer using implementation `hash(into:)` instead of `hashValue`"?

"using implementing" sounds a little strange to me. But I could also be wrong 😄

Reply:
Yeah, you’re right. Probably changed my mind when editing and forgot it there